scripts/local/docker-compose.yml

```
@@ -0,0 +1,43 @@
+    volumes:
+      - mongo_data:/data/db
+    restart: unless-stopped
+
+  redis:
+    image: redis:7.2.4
+    container_name: eatngo-redis
+    ports:
+      - "6379:6379"
+    command: redis-server --requirepass eatngolocal
+    volumes:
+        - redis_data:/data/redis
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)